### PR TITLE
Bump to gnuradio 3.10/3.11

### PR DESCRIPTION
**Pull request by bkerler**
Updated module to gnuradio 3.10/3.11.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +0,0 @@
-gr-cc11xx
-=========
-
-This GNU Radio out of tree module aims at providing a fully functional
```
Comment (andrepuschmann) [inline README.md]: I'd not remove this file, the content is useful and is a a good entry page for github visitors. Alternative is to use the manifest above but this is not automatically picked by github I think

**Review by andrepuschmann — CHANGES_REQUESTED** [review]
Thanks for this port @bkerler . I am not able to test this myself so I am assuming it's tested on your end. The main request I have regarding the PR is the readme. I think a good entry page with some basic information is useful for the visitor of the github page. Let me know what you think. Thanks

File: README.md (hunk 1 above)
Comment (bkerler) [reply README.md]: Sorry, that was by mistake. Of course I added it.

**Comment by andrepuschmann** [comment]
Sorry for the late response and thanks again. Merging

**Pull request merged**